1/debian9/1.16/Dockerfile

```diff
@@ -0,0 +1,45 @@
+# WARNING: This Dockerfile installs NGINX Prometheus Exporter and should be used
```

Comment:
Could you remove the comment? This Dockerfiles doesn't include exporter anymore.

Reply:
Sure. Done